utils/Cargo.toml

```diff
@@ -12,24 +12,25 @@ license = "Apache-2.0"
 [dependencies]
 ark-ec.workspace = true
 ark-ff.workspace = true
-ark-poly.workspace = true 
+ark-poly.workspace = true
 ark-serialize.workspace = true
 bcs.workspace = true
 rayon.workspace = true
 serde.workspace = true
 serde_with.workspace = true
 hex.workspace = true
 num-bigint.workspace = true
-num-integer.workspace = true 
+num-integer.workspace = true
 num-traits.workspace = true
-rmp-serde.workspace = true 
+rmp-serde.workspace = true
 sha2.workspace = true
 thiserror.workspace = true
 rand.workspace = true
 rand_core.workspace = true
+
```

Comment:
![image](https://github.com/user-attachments/assets/36a29269-4351-4226-978d-db5b90596368)

Reply:
This was intentional actually, the idea is to keep in-project dependencies separately from the outer-project ones. Note that we do this quite a lot both in other cargo files, and in rust imports within the codebase too.